readme.md

```diff
@@ -30,18 +30,16 @@ You can use any `@celo/contractkit` version at least as recent as `1.5.1` includ
 
 ## Supported wallets
 
-| Wallet                                                                                     | sendTransaction    | signTransaction    | signTypedData      | signPersonal       |
-| ------------------------------------------------------------------------------------------ | ------------------ | ------------------ | ------------------ | ------------------ |
-| Plaintext private key                                                                      |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |
-| [Ledger](https://www.ledger.com/)                                                          |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |
-| [WalletConnect](https://walletconnect.org/)                                                |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |
-| [Celo Extension Wallet (Metamask fork)](https://github.com/dsrvlabs/celo-extension-wallet) | :white_check_mark: |                    |                    |                    |
-| [Valora](https://valoraapp.com/)                                                           |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |
-| [Celo Wallet](https://celowallet.app/)                                                     |                    |                    |                    |                    |
-| [Celo Terminal](https://celoterminal.com/)                                                 |                    |                    |                    |                    |
-| [MetaMask](https://metamask.io/)                                                           | :white_check_mark: |                    |                    |                    |
-| [Celo Dance](https://celo.dance/)                                                          |                    |                    |                    |                    |
-| [Steakwallet](https://steakwallet.fi/)                                                     |                    |                    |                    |                    |
+- Plaintext private key
+- Ledger
+- WalletConnect
+- Celo
+- Valora
+- Celo
+- Celo
```

Comment:
looks like just "Celo" is listed a few times instead of "celo wallet" celo Terminal, Celo Extenteion etc

Reply:
Oh... 🤦‍♀️ I misused the vscode multiple select lol

Reply:
Thanks for catching it :D